source/virtualBuffers/adobeAcrobat.py

```diff
@@ -120,3 +121,10 @@ def event_valueChange(self, obj, nextHandler):
 			return nextHandler()
 		if not self._handleScrollTo(obj):
 			return nextHandler()
+
+	def _get_ElementsListDialog(self):
+		return ElementsListDialog
+
+class ElementsListDialog(browseMode.ElementsListDialog):
+
+	ELEMENT_TYPES=(browseMode.ElementsListDialog.ELEMENT_TYPES[0],browseMode.ElementsListDialog.ELEMENT_TYPES[1])

```

Comment:
I agree, although the approach I used also is used in
    NVDAObjects.window.winword.ElementsListDialog. If you like, I can
    apply your feedback there as well for this request.